src/app/(auth)/sign-in/page.tsx

```diff
@@ -22,6 +22,10 @@ const Page = async () => {
         <Input type="password" name="password" id="password" />
         <br />
       </AuthForm>
+      <a href="api/oauth2/github" className="buttonClass">
+        Sign in with GitHub
+      </a>
```

Comment:
Is there any styling on this?

Reply:
Not right now , do we have a plan to redo the sign up page or something , cause i might just do it whole there or something?

Reply:
I believe redo the sign up page is not in this PR scope.

Reply:
alright